src/egon/data/datasets/electrical_neighbours.py

```diff
@@ -0,0 +1,1025 @@
+            WHERE country != 'DE'
+            AND scn_name = 'eGon2035')
+        AND scn_name = 'eGon2035'
+        """
+    )
+
+    # Select generators from TYNDP capacities
+    gen = capacities[
+        capacities.carrier.isin(
+            [
+                "other_non_renewable",
+                "wind_offshore",
+                "wind_onshore",
+                "solar",
+                "other_renewable",
+                "reservoir",
+                "run_of_river",
+                "lignite",
+                "coal",
+                "oil",
+                "nuclear",
+            ]
+        )
+    ]
+
+    # Set bus_id
+    gen.loc[
+        gen[gen["Node/Line"].isin(map_buses.keys())].index, "Node/Line"
+    ] = gen.loc[
+        gen[gen["Node/Line"].isin(map_buses.keys())].index, "Node/Line"
+    ].map(
+        map_buses
+    )
+
+    gen.loc[:, "bus"] = (
+        get_foreign_bus_id().loc[gen.loc[:, "Node/Line"]].values
+    )
+
+    # insert data
+    session = sessionmaker(bind=db.engine())()
+    for i, row in gen.iterrows():
+        entry = etrago.EgonPfHvGenerator(
+            scn_name="eGon2035",
+            generator_id=int(db.next_etrago_id("generator")),
+            bus=row.bus,
+            carrier=row.carrier,
+            p_nom=row.cap_2035,
+        )
+
+        session.add(entry)
+        session.commit()
+
+
+def insert_storage(capacities):
+    """Insert storage units for foreign countries based on TYNDP-data
+
+    Parameters
+    ----------
+    capacities : pandas.DataFrame
+        Installed capacities per foreign node and energy carrier
+
+
+    Returns
+    -------
+    None.
+
+    """
+    targets = config.datasets()["electrical_neighbours"]["targets"]
+    map_buses = get_map_buses()
+
+    # Delete existing data
+    db.execute_sql(
+        f"""
+        DELETE FROM {targets['storage']['schema']}.{targets['storage']['table']}
+        WHERE bus IN (
+            SELECT bus_id FROM
+            {targets['buses']['schema']}.{targets['buses']['table']}
+            WHERE country != 'DE'
+            AND scn_name = 'eGon2035')
+        AND scn_name = 'eGon2035'
+        """
+    )
+
+    # Select storage capacities from TYNDP-data
+    store = capacities[capacities.carrier.isin(["battery", "pumped_hydro"])]
+
+    # Set bus_id
+    store.loc[
+        store[store["Node/Line"].isin(map_buses.keys())].index, "Node/Line"
+    ] = store.loc[
+        store[store["Node/Line"].isin(map_buses.keys())].index, "Node/Line"
+    ].map(
+        map_buses
+    )
+
+    store.loc[:, "bus"] = (
+        get_foreign_bus_id().loc[store.loc[:, "Node/Line"]].values
+    )
+
+    # insert data
+    session = sessionmaker(bind=db.engine())()
+    for i, row in store.iterrows():
+        entry = etrago.EgonPfHvStorage(
+            scn_name="eGon2035",
+            storage_id=int(db.next_etrago_id("storage")),
+            bus=row.bus,
+            max_hours=(6 if row.carrier == "battery" else 168),
```

Comment:
max_hours should be taken from scenario_parameters table.

Reply:
For now I just updated the max_hours for pumped hydro storages. As soon as #549 is merged, I will update this.